background/providers/rss/lib/parse-feed.js

```diff
@@ -0,0 +1,120 @@
+import {EventEmitter} from 'events';
+import {promisifyAll} from 'bluebird';
+import xml2js from 'xml2js';
+
+const {parseStringAsync} = promisifyAll(xml2js);
```

Comment:
Let's go with using `util.promisify` until we can find a replacement library. Since we already use babel we shouldn't have any issues with using new features.